packages/dataviews/src/types.ts

```diff
@@ -443,6 +443,11 @@ interface ViewBase {
 	 * The field to group by.
 	 */
 	groupByField?: string;
+
+	/**
+	 * Whether infinite scroll is enabled.
+	 */
+	infiniteScroll?: boolean;
```

Comment:
Optional: `infiniteScroll` reads more like a noun (the feature itself) rather than a boolean state. I'm wondering if it'd aid readability to have a name that indicates a true/false condition?

Then assignments like `const isInfiniteScroll = view.infiniteScroll;` above would be semantically redundant.

E.g., 


```
infiniteScrollEnabled
hasInfiniteScroll
enableInfiniteScroll
infiniteScrollActive
```

Reply:
`infiniteScrollEnabled` is probably the most accurate because that's what the setting checks for.